content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -31,19 +31,19 @@ If you do not specify a referential action, Prisma [uses a default](#referential
 
 <Admonition type="alert">
 
-It is extremely important that you check the [upgrade paths for referential actions](../../../../../guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions) section before you continue. The `referentialActions` feature **removes the safety net in Prisma Client that prevents cascading deletes at runtime**. If you use the feature _without upgrading your database_, the [old default action](../../../../../guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referencial-actions#the-2250-default-referential-actions) - `ON DELETE CASCADE` - becomes active. This may result in cascading deletes that you did not expect.
```

Comment:
intentional that the "old default action" link lost the anchor?

Reply:
Good catch, as per another comment ive now added an id to link to as we removed the anchor for the default values. Will fix